doorstop/run_bepinex.sh

```diff
@@ -5,73 +5,87 @@
 #
 # Usage: Configure the script below and simply run this script when you want to run your game modded.
 
-if [ -z "$1" ]; then echo "Please open run.sh in a text editor and configure executable name. Comment or remove this line when you're done." && exit 1; fi
-
 # -------- SETTINGS --------
 # ---- EDIT AS NEEDED ------
 
 # EDIT THIS: The name of the executable to run
 # LINUX: This is the name of the Unity game executable 
 # MACOS: This is the name of the game app WITHOUT the .app
-executable_name="";
-
+executable_name=""
 
 # The rest is automatically handled by BepInEx
 
 # Whether or not to enable Doorstop. Valid values: TRUE or FALSE
-export DOORSTOP_ENABLE=TRUE;
+export DOORSTOP_ENABLE=TRUE
 
 # What .NET assembly to execute. Valid value is a path to a .NET DLL that mono can execute.
-export DOORSTOP_INVOKE_DLL_PATH=${PWD}/BepInEx/core/BepInEx.Preloader.dll;
-
+export DOORSTOP_INVOKE_DLL_PATH="${PWD}/BepInEx/core/BepInEx.Preloader.dll"
 
 # ----- DO NOT EDIT FROM THIS LINE FORWARD  ------
 # ----- (unless you know what you're doing) ------
 
-# Backup current LD_PRELOAD because it can break `file` when running from Steam
-LD_PRELOAD_BAK=$LD_PRELOAD;
-export LD_PRELOAD="";
+if [ ! -x "$1" -a ! -x "$executable_name" ]; then
+    echo "Please open run.sh in a text editor and configure executable name."
+    exit 1
+fi
 
-doorstop_libs=${PWD}/doorstop_libs;
-arch="";
-executable_path="";
-lib_postfix="";
+doorstop_libs="${PWD}/doorstop_libs"
+arch=""
+executable_path=""
+lib_postfix=""
 
-os_type=`uname -s`;
+os_type=`uname -s`
 case $os_type in
-    Linux*)     executable_path=${PWD}/${executable_name};
-                lib_postfix="so";;
-    Darwin*)    executable_path=${PWD}/${executable_name}.app/Contents/MacOS/${executable_name};
-                lib_postfix="dylib";;
-    *)          echo "Cannot identify OS (got $(uname -s))!";
-                echo "Please create an issue at https://github.com/BepInEx/BepInEx/issues."; 
-                exit 1;;
+    Linux*)
+        executable_path="${PWD}/${executable_name}"
+        lib_postfix="so"
+        ;;
+    Darwin*)
+        executable_path="${PWD}/${executable_name}.app/Contents/MacOS/${executable_name}"
```

Comment:
Naturally, this line should be changed to remove `.app` if `executable_name` were to include `.app` into itself.

Reply:
Small detail, executable_name needs to be without the .app suffix for the last part.